### ISSUE TITLE
PTXPrint goes into endless loop, does not finish nor report an error

### ISSUE THREAD

**Issue by sewhite** [sillsdev/ptx2pdf#279]
First attempt to print created a "try changing your settings" message. I unchecked things like the custom style sheet and PrintDraft changes in Advanced, and succeeded in getting a PDF. I tried adding some of the advanced settings back, and when I clicked print the progress bar went to 1/2, then sat there for minutes until I shut off PTXPrint in Task Manager.

[CUV-DefaultPTXprintArchive.zip](https://github.com/sillsdev/ptx2pdf/files/5465533/CUV-DefaultPTXprintArchive.zip)

**Comment by davidg-sil** [comment]
Certain error paths leave PTXprint unable to do much except save your settings. It's probably better 
that it remains alive rather than crash entirely so you /can/ save your settings. But if it's crashed then it is probably better if you quit and restart. Sorry, I almost certainly don't have time to look at any more issues in detail tonight.

**Comment by davidg-sil** [comment]
I've just tried to replicate your error. Unfortunately I don't have your font... That did seem to replicate your error though! Or at least... sort of.

What I saw is as you described, however, since I started ptxprint from a terminal (command shell, pick your terminology), I saw that what was happening was that XeTeX was asking for input.

I.e. it was not in an endless loop, it was (probably invisibly) waiting for input. After 5mins, PTXprint would have given up waiting and killed the process, I expect.

@markpenny you probably want to close STDIN or whatever it gets called in python, before you 
fork/spawn the XeTeX process. Alternatively, you have these optinons:
```
 -interaction=STRING     set interaction mode (STRING=batchmode/nonstopmode/scrollmode/errorstopmode)
```
I suggest ```nonstopmode``` which will  crash out rather than wait for input. ```batchmode``` also suppresses terminal output. ```errorstopmode``` stops and asks for help.
